src/App.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
```

Comment:
Why did you use javascript on this project instead of typescript?

Reply:
Hi Stefan,

The reasons are:

1- It is not part of the test requirement. I believe the test description only mentions it when it says, `"Prefer TypeScript related backend frameworks"`.

2- I picked JS because I have much more experience using JS than with TypeScript. So I choose the tool that I know best, which gives me a better advantage to achieve better results, as fast as possible.

For your information, I worked only once with TypeSecript in an Angular project.

TypeScript has its values for sure. I believe not every project must use it. Sometimes simple and native tools are better.